### DISCUSSION TITLE
Windows 11, Duplicate Icon

### DISCUSSION THREAD

**Discussion by Wynadorn** [File-New-Project/EarTrumpet#931] [Ideas]
## Description
On windows 11 there's currently two volume icons when using Eartrumpet. I think we should look for a solution or workaround to this.

For example: an invisible button on top of the windows volume button, when clicked it would open the Eartrumpet menu and also intercept the click on the original windows icon. Not ideal of course, but just as an example.

Maybe there's a better solution out there

## Meta
* EarTrumpet version: 2.1.9.0
* Windows 11 version: 22000.194

## Related Issue
#914

**Comment by riverar** [comment]
Moving to Ideas under Discussions. But yep, it's something we're actively investigating. (We're also _lobbying_ Microsoft for change in this area.)

**Comment by m-deutsch** [comment]
Adding an additional icon option to change it to a "mixer" (something like the [MDI tune icon](https://materialdesignicons.com/icon/tune)) instead of the standard volume icon might be another way to approach it.  It would get rid of the duplicate icon while still keeping usability.

Not an ideal end state, but maybe something until Microsoft allows otherwise.

**Reply by riverar** [reply]
@m-deutsch Tip: In Settings, you can turn on our legacy icon.

**Reply by onesounds** [reply]
I found this discussion because this problem. This confuses me. Of course, a legacy icon may be used. but It is difficult to say that legacy icon is a good design. (This is especially true in small sizes such as trays.) I just want to use another icon. I think it would be nice to provide some additional icons (and color) and let them choose. 

Thank you for the good program.

**Comment by Celuni** [comment]
Detour the nasty shell! Like why isn't that done yet??

**Comment by Celuni** [comment]
"Uhh but then Microsoft bans us from the MSSTORE" is not a valid argument xDDD

**Comment by Celuni** [comment]
Left click menu: "Windows.UI.Core.CoreWindow"
Right click menu: Title: "PopupHost" Class: "Xaml_WindowedPopupClass" ExStyle 0x08200008 Style 0x94000000
Hover balloon: "Xaml_WindowedPopupClass"

**Comment by dbareis** [comment]
It would be nice if we could choose our own icon or at least the color for the existing ones as even with the trumpet icons it gets lost amongst all the other white icons. Other than that I'd be happy enough with an orange trump as per the apps icon (at least one I found googling)..

**Comment by DylwinTFTW** [comment]
I know StartAllBack lets you remove the volume icon, so it might be possible.

**Comment by dryya** [comment]
[Explorer Patcher](https://github.com/valinet/ExplorerPatcher) also solves this issue. After installing, I went to its settings -> taskbar -> customize system icons in the notification area to disable the windows volume button and only use EarTrumpet.

**Reply by Kolya-gfx** [reply]
Does this feature still work in Windows 11 22H2 update? Since I have the function to turn off the volume icon from Windows is blocked and I can not access it. How can I solve this?

**Comment by ghost** [comment]
maybe Different Icons for different outputs?

**Comment by m417z** [comment]
A Windhawk user pointed me to this discussion. Hiding the volume icon and other system tray icons was requested by several users since Windows 11 became available. Customizing the system tray is possible with Windhawk via manual styling, but due to the user demand I created a mod which allows to do that via simple options: [Taskbar tray system icon tweaks](https://windhawk.net/mods/taskbar-tray-system-icon-tweaks).

Here's a short demonstration:

https://github.com/user-attachments/assets/3b8f06b5-d6ad-4822-bfb1-6faa86477bdd
